scripts/start_supervisor.py

```diff
@@ -115,7 +117,26 @@ def makeRosInterface():
 
     launchScript(initCode,'initialize SoT')
     ri = makeRosInterface ()
+
+    ## Initialize pose of robot root_joint
+    # read SE(2) motion from ROS parameter
+    moveRootJoint = rospy.get_param ("/robot_initial_pose")
+    x, y, z, X, Y, Z, W = map (float, moveRootJoint.split (' '))
+    # request SoT to publish robot state
+    ri.publishState (Empty)
```

Comment:
Could you add a `\todo` note like
*This is required for simulation and useful for debugging on the robot. As it isn't necessary, as it can be easily turned on (e.g. via a ROS service) and as it cannot be easily turned off, it shouldn't be turned on by default for all running mode.*